Marlin/temperature.cpp

```diff
@@ -1001,14 +1001,11 @@ void Temperature::init() {
 
   #if ENABLED(HEATER_0_USES_MAX6675)
 
-    #if DISABLED(SDSUPPORT)
-      OUT_WRITE(SCK_PIN, LOW);
-      OUT_WRITE(MOSI_PIN, HIGH);
-      SET_INPUT(MISO_PIN);
-      WRITE(MISO_PIN,1);
-    #else
-      OUT_WRITE(SS_PIN, HIGH);
-    #endif
+    OUT_WRITE(SCK_PIN, LOW);
+    OUT_WRITE(MOSI_PIN, HIGH);
+    SET_INPUT(MISO_PIN);
+    WRITE(MISO_PIN, HIGH);
+    OUT_WRITE(SS_PIN, HIGH);
```

Comment:
By test, these all the writing process are needed whether `SDSUPPORT` is enabled or not.
If one of those lack, MINTEMP Err or freeze occures.
I wonder that why `OUT_WRITE(SS_PIN, HIGH)` is also needed when `SDSUPPORT` is disabled.